mkdocs.yml

```diff
@@ -97,7 +97,6 @@ extra:
       link: https://discord.gg/fpFWRQP
       alt: Hackathons UK Discord Server
 
-  placeholder_hackathon_icon: /static/art/hackathon_tiles/default_hackathon_image.png
```

Comment:
looks like this was the only place `default_hackathon_image.png` was used - could remove it unless it's possible it'll be needed in the future?

Reply:
I believe it’s removed as part of this PR, see the diffs 😋

Reply:
missed that, sorry!